src/Admin/AdminHelper.php

```diff
@@ -37,7 +37,7 @@ class AdminHelper
     /**
      * @var PropertyAccessorInterface
      */
-    protected $propertyAccessor;
+    private $propertyAccessor;
```

Comment:
This is a BC break isn't it?

Reply:
Yes but the target branch is master, so that should be fine.

Reply:
OK I am just thinking that we make it internal in master, which would just raise a warning, but making it private in the same step is another user experience 😄

Reply:
Friendly ping @VincentLanglet, thoughts?

Reply:
The class is declared as internal, people shouldnt use it.

Plus, if we dont do this change on new major, we never do.

Reply:
Ok fine